### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sritchie**
This PR adds an updated readme with a clearer (IMO) getting started flow that explains CDN-based and npm-based installations.

See the new README here: https://github.com/unconed/mathbox/blob/sritchie/readme_update/README.md

### CODE REVIEW

**Comment by sritchie** [comment]
@ChristopherChudzicki lmk if you think this is more clear, or if I should change or add anything!

**Comment by sritchie** [comment]
@ChristopherChudzicki this should be good to go for a review now. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,114 +1,142 @@
...
-- [Data/Shape Mapping](http://acko.net/files/mathbox2/iframe-lineup.html)
-- [LaTeX/HTML/GL Labels](http://acko.net/files/mathbox2/iframe-labels.html)
-- [Quaternion Hypersphere](http://acko.net/files/mathbox2/iframe-quat.html)
-- [Render-to-Texture History](http://acko.net/files/mathbox2/iframe-rtt-history.html)
-- [Vertex Warping](http://acko.net/files/mathbox2/iframe-vertex.html)
-- [Volumetric Vectors](http://acko.net/files/mathbox2/iframe-volume.html)
+**Presentations**:
 
-Presentations:
+- [The Pixel Factory](http://acko.net/files/gltalks/pixelfactory/online.html#0**
```
Comment (ChristopherChudzicki) [inline README.md:21]: **Requested Change:**

```suggestion
- [The Pixel Factory](http://acko.net/files/gltalks/pixelfactory/online.html#0)
```

File: README.md [hunk 2]
```diff
@@ -1,114 +1,142 @@
...
-Open the included `/examples` to see more demos.
+### NPM Package
 
----
+- Run the following in your project's directory to install MathBox and
+  [Three.js][three-url] via [npm][npm-url]:
 
-## Docs & Help
+```bash
+npm install --save mathbox three
```
Comment (ChristopherChudzicki) [inline README.md:48]:
```suggestion
npm install mathbox three
```
`--save` is the default https://stackoverflow.com/a/19578808/2747370

File: README.md [hunk 3]
```diff
@@ -1,114 +1,142 @@
...
-- [List of Primitives](docs/primitives.md) for a full element reference.
-- [Writing Custom Shaders](docs/shaders.md) for info on custom shaders and GPU-side processing.
-- [Context API](docs/context.md) for advanced usage.
+```js
+import "mathbox/mathbox.css"
 
-Join us in the [MathBox Google Group](https://groups.google.com/forum/#!forum/mathbox) or #mathbox on Freenode.
+import { * as THREE } from "three"
+import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js"
+import { * as MathBox } from "mathbox"
```
Comment (ChristopherChudzicki) [inline README.md:59]: **Requested Change:**

```suggestion
import * as THREE from "three"
import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js"
import * as MathBox from "mathbox"
```

(`import { * as THREE } from "three"` is invalid js)

File: README.md [hunk 4]
```diff
@@ -1,114 +1,142 @@
...
+  controls: {
+    // Orbit controls, i.e. Euler angles, with gimbal lock
+    klass: THREE.OrbitControls
+  },
+};
+const root = MathBox.mathBox(options);
 ```
 
-See [threestrap](https://github.com/unconed/threestrap) for all available `options`. e.g. To spawn inside a specific element, do:
+> See [threestrap](https://github.com/unconed/threestrap) for all available
```
Comment (ChristopherChudzicki) [inline README.md:108]: ~I don't think this needs to be a block quote.~ Edit: Seems like a reasonably common way to highlight stuff on github.

Comment (ChristopherChudzicki) [reply README.md:108]: Today I learn https://stackoverflow.com/a/72327818/2747370

> **Note**
> This is a note

_Not necessarily suggesting adding the note. Just...sharing weird stuff I learn about github's markdown._

File: README.md [hunk 5]
```diff
@@ -125,23 +153,94 @@ const view = mathbox
...
+
+## Who's using MathBox?
 
----
+- [Math3D online graphing calculator](https://www.math3d.org/)
+- [KineticGraphs JS Engine](https://kineticgraphs.org/) ([code](https://github.com/cmakler/kgjs))
+- [Textbook: "Interactive Linear Algebra""](https://textbooks.math.gatech.edu/ila/) ([code](https://github.com/QBobWatson/ila))
+- Many visualizations at [Sam Zhang's homepage](https://sam.zhang.fyi/#projects)
+- Jesse Bettencourt's [Torus Knot Fibration](http://jessebett.com/TorusKnotFibration/) Master's project ([code](https://github.com/jessebett/TorusKnotFibration))
+- [Interactive knot visualizations](https://rockey-math.github.io/mathbox/graph3d-curve)
```
Comment (ChristopherChudzicki) [inline README.md:222]: Any of your projects you'd like to add here?

File: README.md [hunk 6]
```diff
@@ -125,23 +153,94 @@ const view = mathbox
...
-MathBox and ShaderGraph (c) Steven Wittens 2013-2022. MIT License.
+And the many demos listed on [this
+thread](https://groups.google.com/g/mathbox/c/Uvyb5fHaLq4) of the [MathBox
+Google group][google-group-url].
+
+## License
+
+[MIT License](LICENSE.md).
+
+MathBox and ShaderGraph (c) Steven Wittens 2013-2022.
```
Comment (ChristopherChudzicki) [inline README.md:232]:
```suggestion
MathBox and ShaderGraph (c) Steven Wittens 2013-2023.
```

**Review by ChristopherChudzicki — CHANGES_REQUESTED** [review]
Looks good! Noticed some minor typos.

File: README.md (hunk 3 above)
Comment (sritchie) [reply README.md:59]: Whoops, clearly I was going from memory here :)

File: README.md (hunk 4 above)
Comment (sritchie) [reply README.md:108]: that is great! These callouts are one of the reasons I've used asciidoc in other projects. I'll make this change...

File: README.md (hunk 5 above)
Comment (sritchie) [reply README.md:222]: Once I get them up to snuff I'll add them as I go.

**Comment by sritchie** [comment]
Thanks for the careful read, I took all suggestions.

**Comment by sritchie** [comment]
@ChristopherChudzicki wdyt of a GitHub Action to publish your storybook for mathbox-react to github pages? That would be a nice way to get a README up for mathbox-react...

**Pull request merged**